doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -467,23 +467,64 @@ destroy all backtrack and return values.
 Backtracking
 ++++++++++++
 
-In Ltac2, we have the following backtracking primitives, defined in the
-`Control` module::
+Ltac2 models backtracking computations using streams of values: the head of a non-empty stream represent the first path of computation while its tail stands for the alternative paths of computation when backtracking occurs. The empty stream holding no values represent failure and initiates backtracking.
+
+Backtracking failures (empty streams) are moreover decorated with exceptions to inform the reason of failure. These exceptions are then passed to the tail of a non-empty backtracking computation. The following Ltac2 type summarizes the model for backtracking computations:
```

Comment:
this paragraph is in bold in the refman is that intended? I think it's because there's no empty line between it and the `.. rocqtop::` directive.

Reply:
fixed, thanks for the indication, space sensitivity without proper feedback is getting me crazy...